<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comportComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectRadioLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connect Radio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshComportsBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="rxToneLcdNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="rxToneLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RX Tone:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rxKHzLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startRxBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>START RX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTxBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>START TX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startIdleBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>IDLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startVcoCalBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>START VCO CAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>MODE:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentModeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="rxDataLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RX Data:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rxDataHexLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QComboBox" name="specAnComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>192.168.50.3</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="connectSpecAnBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectRadioLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connect Spectrum Analyzer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="specAnGroupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Spectrum Analyzer</string>
    </property>
    <widget class="QLabel" name="specAnIdnLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="specAnPeakFreqLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Peak Freq. (MHz)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="specAnPeakFreqLCDNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="specAnStartFreqLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Freq. (MHz)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="setVcoVoltageBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vcoVoltageLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.65</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vcoVoltageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>VCO Voltage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="radioStateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Radio State:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="rxDataPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txDataPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rxTextAreaLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>RX DATA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txTextAreaLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TX DATA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="radioSummaryLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NO DATA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
